package/kernel/qca-ssdk/Makefile

```diff
@@ -48,7 +48,7 @@ MAKE_FLAGS+= \
 	$(LNX_CONFIG_OPTS)
 
 ifeq ($(CONFIG_TARGET_SUBTARGET), "ipq807x")
-    MAKE_FLAGS+= CHIP_TYPE=HPPE
+    MAKE_FLAGS+= CHIP_TYPE=HPPE IN_MALIBU_PHY=FALSE
```

Comment:
I would move this to the rest of make flags as otherwise we will have to disable it in ipq60xx, then ipq50xx etc